test/config/ocsp-updater.json

```diff
@@ -5,7 +5,7 @@
     "oldOCSPWindow": "2s",
     "oldOCSPBatchSize": 5000,
     "parallelGenerateOCSPRequests": 10,
-    "ocspMinTimeToExpiry": "72h",
+    "ocspMinTimeToExpiry": "6s",
```

Comment:
Not updating this in config-next as well?

Reply:
Oops, I updated this for manual testing and forgot to revert. Did that now.